src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -168,6 +168,7 @@ From any markdown element, access the menu on the upper right corner to edit or
 * Expand your chart to full screen.
 * [Share your chart](#dash-export) as an image or with a link.
 * Copy the chart to any dashboard.
+* For table charts only, export as a `.csv` file. This can be later used in other applications such as Microsoft Excel.  
```

Comment:
All changes look good technically! Small nitpick: Can we get more active voice and instructive here? e.g.

> For table charts only, export as a `.csv` file. You can import this file into other apps like Microsoft Excel or Google Sheets to do further analysis.

Just making something up, but wondering if we can call out use cases a little more explicitly.

Reply:
Thanks @austin-schaefer ! Made the changes, is it ok to merge?